test/integration/simple_example/simple_example_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package primary_instance_adb_cluster
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSimpleExample(t *testing.T) {
+	example := tft.NewTFBlueprintTest(t)
+
+	example.DefineVerify(func(assert *assert.Assertions) {
+		// Getting the expected Project ID from the outputs
+		projectId := example.GetStringOutput("project_id")
+		// Router Information
+		routerName := example.GetStringOutput("router_name")
+		routerRegion := example.GetStringOutput("router_region")
+		fmt.Printf("%s, %s, %s", projectId, routerName, routerRegion)
+
+		getRouterName := gcloud.Run(t, fmt.Sprintf("compute routers describe %s  --project %s  --region %s ", routerName, projectId, routerRegion))
```

Comment:
nit: prefer Runf here and throughout
https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test@v0.7.0/pkg/gcloud#Runf
```suggestion
		getRouterName := gcloud.Runf(t,"compute routers describe %s  --project %s  --region %s ", routerName, projectId, routerRegion)
```